funcx_sdk/funcx/sdk/asynchronous/funcx_task.py

```diff
@@ -0,0 +1,28 @@
+import asyncio
+
+
+class FuncXTask:
```

Comment:
We should probably extend the asycio future class here with our attributes (task_id) to it. It would be useful to keep that interface unchanged for compatibility.